chain-storage-sqlite-old/src/lib.rs

```diff
@@ -69,14 +71,12 @@ where
             )
             .unwrap();
 
-        /*
         connection
-            .execute("pragma synchronous = off", rusqlite::NO_PARAMS)
+            .execute_batch("pragma journal_mode = WAL")
             .unwrap();
-        */
 
         connection
-            .execute_batch("pragma journal_mode = WAL")
+            .execute_batch("pragma read_uncommitted = true")
```

Comment:
- Why does this need to be done twice?
- Can this result in inconsistent reads when there is an intervening write, or are our append-only records immune to this kind of problem?

Reply:
1. My bad, will remove the second one.
2. Yes, in theory something can go wrong. But since we don't alter or remove records, everything should be fine.

Reply:
Please leave a comment noting this assumption near the pragma query.

I guess some future trouble may be introduced with queries that select blocks in the current epoch, but with a where that does not address specific blocks.

Is there no way to use the [unlock_notify API](https://www.sqlite.org/unlock_notify.html)? That could be a subject for a future enhancement PR, though.

Reply:
Not sure about that. This may require adding changes to `rusqlite`.

Reply:
Well, everything was way more easier. `rusqlite` already has support for `unlock_notify` and it is working fine.